src/status_im2/contexts/chat/messages/list/style.cljs

```diff
@@ -0,0 +1,83 @@
+(ns status-im2.contexts.chat.messages.list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]
+            [react-native.reanimated :as reanimated]))
+
+(defonce ^:const cover-height 168)
+(defonce ^:const overscroll-cover-height 2000)
+(defonce ^:const cover-bg-color "#2A799B33")
```

Comment:
This is a customization color with 20% opacity. Instead of hard-coding the value here, shouldn't we consider using it from the `colors` directly?